lib/server.js

```diff
@@ -107,7 +107,7 @@ function Server (torrent, opts) {
         return serveIndexPage()
       }
 
-      var index = Number(pathname.slice(1))
+      var index = Number(pathname.split('/')[0])
```

Comment:
Did you actually test this out? This looks wrong. Shouldn't it be `pathname.split('/')[1]`?